tox.ini

```diff
@@ -193,7 +199,7 @@ max-line-length = 88
 python =
   2.6: py26,coveralls,custom
   2.7: py27,coveralls,flake8,pylint,custom
-  3.6: py36,coveralls,black,yamllint,custom
+  3.6: py36,coveralls,black,yamllint,shellcheck,custom
```

Comment:
Does this mean we only run shellcheck on python3.6? If so, why?

Reply:
shellcheck isn't really a python thing - we only need to run it once, no matter what the python version is - we're just using tox as a convenient task runner - same with yamllint